.buildkite/pipeline.yml

```diff
@@ -17,6 +17,20 @@ steps:
       - "queue=runtime"
     artifact_paths:
       - "build_id/**"
+  - label: "Build chromium - Linux"
+    key: "build-chromium-linux"
+    plugins:
+      - thedyrt/skip-checkout#v0.1.1: ~
+    command: "cd /Users/administrator/chromium/src && node buildMac.mjs &&  node replay_build_scripts/upload_build_artifacts.mjs"
+    env:
+      GOMA_SERVER_HOST: simpsonite.goma.engflow.com
+      GOMACTL_USE_PROXY: false
+    agents:
+      - "runtimeType=chromiumbuild"
+      - "os=macos"
+      - "queue=runtime"
+    artifact_paths:
```

Comment:
Lets remove this for now--I think this will race with the linux build, potentially overwriting the build_id stored there.  I think we'll probably just do something like output a build_id_linux and build_id_macos artifact to avoid this....